### DISCUSSION TITLE
[Feature] Add Conditional Album Creation and Auto-Import Functionality

### DISCUSSION THREAD

**Discussion by LoCCai** [immich-app/immich#14883] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Body:

Hi Immich Team,

I hope this message finds you well. I'd like to propose a feature that I believe would be incredibly useful for many users: the ability to create albums that automatically import photos based on user-defined conditions.

Why This Feature Matters
Automation is Key: Many programmers and tech enthusiasts embrace automation as a way to simplify and enhance workflows. Having conditional albums aligns perfectly with this mindset.
Ease of Use: When creating an album, it would be great if users could simultaneously define conditions to automatically fetch relevant photos into the album, making it more convenient and user-friendly.
Suggested Functionality
Create Conditional Albums: Users can define conditions when creating an album. Examples of such conditions include:
File paths that include or exclude specific strings.
Photos or videos containing specific faces (face recognition).
Images captured by a specific camera, device, or lens.
Photos taken during a particular time period.
Flexible Logic: Users should be able to combine conditions using logical operators like OR and AND to configure highly customized content rules.
Scheduled Updates: A periodic check should be implemented to automatically scan for new photos and update the albums accordingly.
This feature would significantly enhance Immich’s capability to organize and manage photos, providing an effortless and personalized experience for users.

Thank you for considering this request. Please let me know if you need further details or clarification.

Best regards

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by danieldietzler** [comment]
This is already covered by "workflows" on our roadmap; https://immich.app/roadmap :)

**Reply by LoCCai** [reply]
good for it

**Reply by sgofferj** [reply]
@danieldietzler 
Is there a Github issue or discussion for that one could subscribe to?

**Reply by danieldietzler** [reply]
Currently there is not, sorry. @sgofferj

**Reply by bo0tzz** [reply]
Yes there is, #1673
